allure-python-commons/src/utils.py

```diff
@@ -323,19 +242,25 @@ def func_parameters(func, *args, **kwargs):
         args_dict.pop(arg_spec.args[0], None)
 
     if kwargs:
-        if sys.version_info < (3, 6):
+        if sys.version_info < (3, 7):
```

Comment:
Dict key order preservation is an implementation detail in CPython 3.6 and it`s not advised to rely upon it. It became a part of the spec in python 3.7